package/features

```diff
@@ -1,37 +1,67 @@
-nodefault 'web-wizard'
+-- The _ prefix allows to perform boolean logic on feature names
+--
+-- Do not use the 'nodefault' flag in feature specifications that  have
+-- such a boolean match expression rather than a single feature name
```

Comment:
I think a pointer to the documentation would be great here. Without context, the nodefault flag is not self-explanatory.

Reply:
Ah, I forgot we already have a doc page about the feature spec format, I'll have to update that as well.